.github/workflows/velox_backend_arm.yml

```diff
@@ -146,8 +146,8 @@ jobs:
             --extra-conf=spark.gluten.ras.enabled=true 
 
   cpp-test-udf-test:
-    runs-on: ubuntu-24.04-arm
-    container: apache/gluten:centos-8-jdk8
+    runs-on: ubuntu-22.04-arm
```

Comment:
this is a workaround for issue:
https://github.com/apache/incubator-gluten/issues/11063